python/eos/analysis.py

```diff
@@ -217,11 +217,15 @@ def optimize(self, start_point=None, **kwargs):
         """
         Optimize the log(posterior) and returns a best-fit-point summary.
 
-        :param start_point: Parameter point from which to start the optimization, with the elements in the same order as in eos.Analysis.varied_parameters. If not specified, optimization starts at the current parameter point.
+        :param start_point: Parameter point from which to start the optimization, with the elements in the same order as in eos.Analysis.varied_parameters.
+                            If set to "Random", optimization starts at the random point in the space of the priors.
+                            If not specified, optimization starts at the current parameter point.
         :param start_point: iterable, optional
         """
         if start_point == None:
             start_point = [float(p) for p in self.varied_parameters]
+        elif start_point == "Random":
```

Comment:
See above. This should now also be adopted for the ``eos-analysis`` script. I think that will require a little bit of modifcations, such that either a list or a string are accepted as the start point.

Reply:
I added a new option, let me know if this looks ok